packages/block-editor/src/index.js

```diff
@@ -13,5 +12,5 @@ import '@wordpress/notices';
 import './hooks';
 export * from './components';
 export * from './utils';
-export { storeConfig } from './store';
+export { storeConfig, storeDefinition } from './store';
```

Comment:
@youknowriad, do you know why we expose `storeConfig` as part of public API? It looks like a good opportunity to consolidate it under `storeDefinition`.

Reply:
yes, in my imagination these are the same thing but maybe we need to keep both here for BC reasons even after the proposal I made. We need it to create "sub registries" (reusable block I believe).

Reply:
I know some packages do export `storeConfig` to make it accessible in tests - it could be done without making it public API though.

Reply:
We should keep it in mind when improving `createStoreDefintion`.